npm/vue/cypress/component/setup/HelloWorld.cy.ts

```diff
@@ -24,4 +24,15 @@ describe('HelloWorld', () => {
     .click()
     .should('contain', '1')
   })
+
+  it('exposes hello', () => {
```

Comment:
Should this be named more like "exposes `count` from `defineExpose`"?

Reply:
Fixed